### PR TITLE
Doc change. FROM FINAL can be used not only with CollapsingMergeTree

### PR DESCRIPTION
**Pull request by den-crane**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

### CODE REVIEW

**Comment by blinkov** [comment]
хорошо б их ещё ссылками на страницы про движки сделать

File: docs/en/query_language/select.md [hunk 1]
```diff
@@ -108,7 +108,7 @@ A table function may be specified instead of a table. For more information, see
...
-The FINAL modifier can be used only for a SELECT from a CollapsingMergeTree table. When you specify FINAL, data is selected fully "collapsed". Keep in mind that using FINAL leads to a selection that includes columns related to the primary key, in addition to the columns specified in the SELECT. Additionally, the query will be executed in a single stream, and data will be merged during query execution. This means that when using FINAL, the query is processed more slowly. In most cases, you should avoid using FINAL. For more information, see the section "CollapsingMergeTree engine".
+The FINAL modifier can be used only for a SELECT from ReplacingMergeTree, SummingMergeTree, AggregatingMergeTree, CollapsingMergeTree and VersionedCollapsingMergeTree tables. When you specify FINAL, data is selected fully "merged". Keep in mind that using FINAL leads to a selection that includes columns related to the primary key, in addition to the columns specified in the SELECT. Additionally, the query will be executed in a single stream, and data will be merged during query execution. This means that when using FINAL, the query is processed more slowly. In most cases, you should avoid using FINAL.
```
Comment (alexey-milovidov) [inline docs/en/query_language/select.md]: Did you forget GraphiteMergeTree?

PS. You can write - all MergeTree engines with non-trivial merge algorithm.

Comment (den-crane) [reply docs/en/query_language/select.md]: I did not. I double checked.

SELECT *
FROM test
FINAL

Received exception from server (version 19.14.1):
Code: 181. DB::Exception: Received from localhost:9000. DB::Exception: Storage GraphiteMergeTree doesn't support FINAL.

CREATE TABLE test
(
    `Path` String,
    `Time` DateTime,
    `Value` Int64,
    `Version` Int64
)
ENGINE = GraphiteMergeTree('graphite_rollup_example')
ORDER BY tuple()

Comment (den-crane) [reply docs/en/query_language/select.md]: I can write all  MergeTree engines except MergeTree and GraphiteMergeTree

Comment (alexey-milovidov) [reply docs/en/query_language/select.md]: Ok. No need for it.

**Pull request merged**